src/controllers/profileController.js

```diff
@@ -0,0 +1,60 @@
+import * as profileService from '../services/profileService';
+import * as ApplicationError from '../utils/errors/applicationsErrors';
+import * as profileValidations from '../validations/profileValidation';
+import * as notFound from '../utils/errors/notFoundError';
+
+const updateProfile = async (userId, requestBody, response, next) => {
+  try {
```

Comment:
@SergeNm I can’t really test your endpoints while the logic is not handled well.

Reply:
The logic is to create a profile while registering. Please firstly register a user.